src/middlewares/task.middleware.js

```diff
@@ -221,7 +224,43 @@ export default (store: any) => (next: any) => (action: any) => {
             `\u001b[31;1m${abortMessage}\u001b[0m`
           )
         );
-      });
+      } else {
+        // Our child was spawned using `shell: true` to get around a quirk with
+        // electron not working when specifying environment variables the
+        // "correct" way (see comment above).
```

Comment:
This isn't actually true anymore!

Can you please add a TODO after this paragraph, like this:

```
// TODO: Because we no longer use 'shell: true', this is probably not necessary anymore. Consider reverting.
```

Reply:
I'll modify the comment so that we're checking this later.